data/map planets.txt

```diff
@@ -736,6 +736,15 @@ planet "Cool Forest"
 	spaceport `The spaceport here is a major city, with a continuous stream of cargo transports bringing goods manufactured on Cool Forest's sister planet, Diligent Hand. Here, some final assembly, testing, and repackaging is performed before those goods are shipped out to the rest of Saryd space and beyond.`
 	security 0.3
 
+planet "Copperline Station"
+	attributes pirate station frontier "core pirate" smuggling
+	landscape land/distillery1
+	description `C2H5OH: Sweet, sweet ethyl alcohol, the nectar of the Gods spilled upon the heavens themselves. The Great Molecular Cloud which protrudes into this star system has long been a mythical promised land to spacers, whispered of since humanity first turned their spectroscopes towards the stars.`
```

Comment:
Pretty much every other planetary description mentions the name of the planet in the first sentence. This one doesn't mention it until the end of the second paragraph, and instead starts off with a convoluted formula. It doesn't help that ES doesn't have subscript, of course, but it takes a bit of thinking to figure out that that is in fact the chemical formula for ethyl alcohol. Then, the second sentence shifts topic after leaving the subject of the first one without a verb, which makes it read like a title. The whole paragraph takes a lot of reading between the lines, and it took two reads to fully get.

Overall, this paragraph is great storytelling, but I'd love to see some mention of the station's name (maybe why it's called Copperline?) and perhaps the contents of the first sentence can be redistributed among the later ones in this paragraph and the next one?